client/src/components/ManageAccount/ManageAccount.js

```diff
@@ -36,6 +36,12 @@ const ManageAccountView = () => {
       bodyClassName="has-loaded-account"
       welcomeMessageName="ACCOUNT"
       header="Manage Account"
+      messages={
+        !isLoading &&
+        integrations[0].error_message && (
+          <Alert color="danger">{integrations[0].error_message}</Alert>
```

Comment:
Please use `<SectionMessage type="...">` component with type `error` or `warning`.

<details>
<summary>Reference</summary>

- knowledge of the possible errors
- [UI - Components - Message](https://confluence.tacc.utexas.edu/x/gYCeBw)
    > __Warning__: Text that notifies user of an unsuccessful event that _could prevent_ functionality.
    > __Error__: Text that notifies user of an unsuccessful event that _prevents_ functionality.

</details>

If there is concern with doing this piecemeal given existing `<Alert>` components in this file, then the ticket [FP-1116](https://jira.tacc.utexas.edu/browse/FP-1116) can update them all.